lib/commands/clean.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+var path    = require('path');
+var Command = require('../models/command');
+var Promise = require('../ext/promise');
+
+module.exports = Command.extend({
+  name: 'clean',
+  description: 'Cleans up installations, flushes caches and removes build results(tmp/ and dist/ by default).',
```

Comment:
As for beginning of phrase - yes.
I use "by default" because at least dist/ path is configurable